decidim-admin/app/policies/decidim/admin/participatory_process_policy.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+module Decidim
```

Comment:
I am not sure I like how `Pundit` deals with permissions. I feel like `cancancan`'s approach might be simpler.

In `cancancan`, you define `abilities` for each user, and each `ability` lies in a separate class. A user can have many abilities (`Moderator`, `Official`, `Verified`). You then check a particular permission against all abilities to figure out what you can and cannot do.

`Pundit` seems to do it the other way around by putting all possible permissions in the same place. I feel like it provides bigger coupling, although I like how the feature and all its authorization logic are self-contained (otherwise, you would have to scatter its permissions all the way around on the `Ability` classes).

This is actually just a note for the future. Let's see how the authorization codebase evolves around this as we keep adding features.